routes/relationship.js

```diff
@@ -1,152 +1,202 @@
     });
-
-    // Send JSON with new changes
-    res.send(json);
 });
 
-
 /**
  * Delete endpoint, which takes the relationship id.
  */
-router.delete('/delete', function (req, res, next) {
-    // Grab the parameters from the request body that we need
-    const {id} = req.body;
-
-    // Ensure id is specified
-    if (!id) {
-        res.status(400).send("Must specify id!");
-        return;
-    }
-
-    // This assumes the file already exists (and it should)
-    let json = JSON.parse(fs.readFileSync(__dirname + "/../public/relationships.json", 'utf-8'));
-
-    // Remove by id from json
-    // This also removes duplicate instances
-    json = json.filter(function (e) {
-        return e.id !== id;
-    });
-
-    // Write new changes to file
-    fs.writeFile(__dirname + "/../public/relationships.json", JSON.stringify(json), function (err) {
-        if (err) {
-            res.status(400).send("Issue writing file!");
-        }
+router.delete("/delete", function (req, res, next) {
+  // Grab the parameters from the request body that we need
+  const { id } = req.body;
+
+  // Ensure id is specified
+  if (!id) {
+    res.status(400).send("Must specify id!");
+    return;
+  }
+  /*
+  var id = new require("mongodb").ObjectID(id); //req.params.id
+  db.collection("users")
+    .findOne({ _id: id })
+    .then(function (doc) {
+      if (!doc) throw new Error("No record found.");
+      console.log(doc); //else case
     });
+*/
+
+  const database = client.db("newMongoDB");
+  const collection = database.collection("relationships");
+  client.connect().then((response) => {
+    const found = false;
+    //compares id to document id's
+    collection
+      .findById({ id })
+      .then((doc) => {
+        console.log(doc);
+      })
+      .catch((err) => {
+        console.log(err);
+      });
+  });
+
+  /*
+  // This assumes the file already exists (and it should)
+  let json = JSON.parse(
+    fs.readFileSync(__dirname + "/../public/relationships.json", "utf-8")
+  );
+
+  // Remove by id from json
+  // This also removes duplicate instances
+  json = json.filter(function (e) {
+    return e.id !== id;
+  });
+
+  // Write new changes to file
+  fs.writeFile(
+    __dirname + "/../public/relationships.json",
+    JSON.stringify(json),
+    function (err) {
+      if (err) {
+        res.status(400).send("Issue writing file!");
+      }
+    }
+  );
 
-    // Send JSON with new changes
-    res.send(json);
+  // Send JSON with new changes
+  res.send(json);
+  */
 });
 
 /**
  * Update endpoint, which takes the relationship id and relationship data to update.
  */
-router.put('/update', function (req, res, next) {
-    // Grab the parameters from the request body that we need
-    const {id, relationship} = req.body;
-
-    // Ensure id is specified
-    if (!id || !relationship) {
-        res.status(400).send("Must specify id and relationship!");
-        return;
+router.put("/update", function (req, res, next) {
```

Comment:
It looks like this function still uses the relationships.json file instead of connecting to Mongo